packages/esm-patient-registration-app/src/patient-registration/form-manager.ts

```diff
@@ -363,14 +363,6 @@ export class FormManager {
       }
     }
 
-    if (values.unidentifiedPatient) {
```

Comment:
This attribute should still be set if the patient's name is unknown... That's still the case, right?

Reply:
To be clear, this attribute is supposed to be the source of truth for whether we have an anonymous patient or not. That way we're not treating the string "UNKNOWN" as a special value because we would like to be able to localise that string.

Reply:
Yes @ibacher !
This will be set after the person attribute `Unknown patient` and will override the frontend config in case the patient is unknown.